packages/web-components/.storybook/container.scss

```diff
@@ -15,10 +15,12 @@
 
 @use '@carbon/ibmdotcom-styles/scss/internal/callout/tokens' as callout-tokens;
 @use '@carbon/styles/scss/components/button/tokens' as button-tokens;
+@use '@carbon/ibmdotcom-styles/scss/components/leadspace' as leadspace;
```

Comment:
wondering if we can seperate out the tokens like callout out does in separate file, so they only need to import those if they use a different theme. and then also add it to the dotcom [themes cdn file ](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/feat/carbon-for-ibm-dotcom-v2/packages/web-components/src/globals/scss/themes.scss)